mapbox_tilesets/utils.py

```diff
@@ -103,3 +108,59 @@ def validate_geojson(feature):
     }
 
     return validate(instance=feature, schema=schema)
+
+
+def _convert_precision_to_zoom(precision):
+    """Converts precision to zoom level based on the minimum zoom"""
+    if precision == "10m":
+        return 6
+    elif precision == "1m":
+        return 11
+    elif precision == "30cm":
+        return 14
+    else:
+        return 17
+
+
+def _tile2lng(tile_x, zoom):
+    """Returns tile longitude"""
+    return ((tile_x / 2 ** zoom) * 360.0) - 180.0
+
+
+def _tile2lat(tile_y, zoom):
+    """Returns tile latitude"""
+    n = np.pi - 2 * np.pi * tile_y / 2 ** zoom
+    return (180.0 / np.pi) * np.arctan(0.5 * (np.exp(n) - np.exp(-n)))
+
+
+def _calculate_tiles_area(tiles):
```

Comment:
do we want to rename this to make clear it only returns the area for a single tile?